### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rondeau79**

### CODE REVIEW

**Comment by Rondeau79** [comment]
Requesting pull and write access.

**Comment by gwolf2u** [comment]
You shouldn't modify the readme file
You'll need to modify update.xml and create a file changelog_device.txt
Please correct pull request

**Review by gwolf2u — CHANGES_REQUESTED** [review]
Needs changes

**Comment by Rondeau79** [comment]
Requested changes made.

File: README.md [hunk 1]
```diff
@@ -7,24 +7,24 @@ In order for a device to be OTA compliant, there are a few things to know.
...
-        <DeviceCode>
-            <maintainer>MaintainerName</maintainer>
-            <devicename>DeviceName</devicename>
-            <filename>FileName</filename>
-            <buildtype>BuildType</buildtype>
-            <download>DownloadLink</download>
-            <changelog>ChangelogLink</changelog>
-            <gapps>GappsLink</gapps>
-            <forum>ForumLink</forum>
+    <manufacturer id="Motorola">
```
Comment (gwolf2u) [inline README.md:10]: drop changes for readme.md

File: update.xml [hunk 2]
```diff
@@ -276,7 +276,26 @@
     <!-- LYF -->
 
     <!-- MOTOROLA -->
-    <manufacturer id="Motorola"> 
+    	 <manufacturer id="Motorola">
```
Comment (gwolf2u) [inline update.xml:279]: manufacturer Motorola exists, don't add a new one
add your device lower in the file

File: update.xml [hunk 3]
```diff
@@ -276,7 +276,26 @@
...
+            <gapps>https://opengapps.org</gapps>
+            <forumhttps://forum.xda-developers.com/g7-power/development/rom-t4005739></forum>
+            <firmware>FirmwareLink</firmware>
+            <modem>ModemLink</modem>
+            <bootloader>BootloaderLink</bootloader>
+            <recovery>RecoveryLink</recovery>
+            <paypal>PaypalLink</paypal>
+            <telegram>TelegramLink</telegram>
+        </Ocean>
+    </Motorola>
```
Comment (gwolf2u) [inline update.xml:296]: this needs removal

**Review by gwolf2u — CHANGES_REQUESTED** [review]
also make sure to validate your XML if you are not sure how to do it 
a good online validator is this 
https://www.liquid-technologies.com/online-xml-validator

**Comment by Rondeau79** [comment]
Everything checked and lined up to the rest of the devices in the xml.

**Comment by Rondeau79** [comment]
Revert

**Pull request closed without merging**